core-bundle/contao/drivers/DC_Table.php

```diff
@@ -4025,28 +4026,38 @@ protected function treeView()
 		// Show paste button only if there are no root records specified
 		if ($blnClipboard && ($GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['mode'] ?? null) == self::MODE_TREE && $this->rootPaste && Input::get('act') != 'select')
 		{
+			$operations = System::getContainer()->get('contao.data_container.operations_builder')->initialize();
+
 			// Call paste_button_callback (&$dc, $row, $table, $cr, $children, $previous, $next)
 			if (\is_array($GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['paste_button_callback'] ?? null))
 			{
 				$strClass = $GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['paste_button_callback'][0];
 				$strMethod = $GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['paste_button_callback'][1];
 
-				$_buttons = System::importStatic($strClass)->$strMethod($this, array('id'=>0), $table, false, $arrClipboard);
+				$operations->append(array('html'=>System::importStatic($strClass)->$strMethod($this, array('id'=>0), $table, false, $arrClipboard)));
```

Comment:
<img width="458" alt="" src="https://github.com/user-attachments/assets/ccff07d6-555f-42bb-93fe-8d8a11244150">

That is because `$operations` is not initialized in the `if` condition on line 4423.

Reply:
uhm, but it is initialized on line 4029?

Reply:
Sorry, I chose the wrong line to add this. The problem is in the `if` condition on line 4423.

Reply:
This cannot happen though. Line 4418 is executed if `Input::get('act') == 'select'` and 4441 only if `Input::get('act') != 'select'` 🤔

Reply:
It is however incorrectly appended to `$_buttons`, that's fixed in 7fdd3f9e9187000fa826c373ffb2eca4a09b8f1e